src/main/groovy/org/mockito/release/notes/format/DetailedFormatter.java

```diff
@@ -53,13 +53,20 @@ public String formatReleaseNotes(Collection<ReleaseNotesData> data) {
         return sb.toString().trim();
     }
 
-    static String releaseSummary(Date date, ContributionSet contributions, Map<String, Contributor> contributors,
+    static String releaseSummary(String version, Date date, ContributionSet contributions, Map<String, Contributor> contributors,
                                  String vcsCommitsLink, String publicationRepository) {
         return authorsSummary(contributions, contributors, vcsCommitsLink) +
-                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + publicationRepository + "\n" +
+                " - *" + DateFormat.formatDate(date) + "*" + " - published to " + getBintrayBadge(version, publicationRepository) + "\n" +
```

Comment:
Release notes formatter is not aware where the release is synced to. I think we can merge this code and start thinking how to model Maven Central release better.